posthog/models/integration.py

```diff
@@ -117,7 +117,8 @@ def validate_request(cls, request: Request):
         if not slack_config["SLACK_APP_SIGNING_SECRET"] or not slack_signature or not slack_time:
             raise SlackIntegrationError("Invalid")
 
-        if time.time() - slack_time > 60 * 5:
+        # Check the token is not older than 5mins
+        if time.time() - float(slack_time) > 300:
```

Comment:
Is it safe if this throws? 

If `slack_time` won't convert to a float does validation still fail?

Reply:
Yeah. Yeah..... Maybe... I'll add a test!

Reply:
Added a test and now we have a non-500 response which is better